core/src/main/java/com/vml/es/aem/acm/core/code/Condition.java

```diff
@@ -303,6 +305,30 @@ public LocalTime dayEndTime() {
         return LocalTime.of(23, 59, 59, 999999999);
     }
 
+    // Date based
+
+    public boolean isDate(String dateString) {
+        LocalDateTime localDateTime = DateUtils.localDateTimeFromString(dateString);
+        return isDate(localDateTime);
+    }
+
+    public boolean isDate(ZonedDateTime zonedDateTime) {
+        LocalDateTime localDateTime =
+                zonedDateTime.withZoneSameLocal(ZoneOffset.systemDefault()).toLocalDateTime();
```

Comment:
hmm in `State/InstanceSettings` we are returning also timezone; I feel like we should define server timezone in one place in code, then reference twice; there and from here